File: Javascript/Matrices y arreglos/matriz/matriz.js
// implementacion en JavaScript
const twoDimensionalArray = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

console.log("Los elementos de la matriz son:");
for (const row of twoDimensionalArray) {
    for (const element of row) {
        process.stdout.write(element + " "); // mostrando los elementos de la fila separados por espacios especiales
        console.log(); // salto de linea para mostrar los elementos de la siguiente fila
    }
}

console.log("Los elementos de la matriz en forma de lista son:");
for (const row of twoDimensionalArray) {
    console.log(row); // mostrando los elementos de la fila como una lista
}

File: C#/Matrices y Arreglos/matriz/Matriz.cs
using System;

// implementacion en C#
class Matriz
{
    static void Main(string[] args)
    {
        int[,] twoDimensionalArray = {
            { 1, 2, 3 },
            { 4, 5, 6 },
            { 7, 8, 9 }
        };

        Console.WriteLine("Los elementos de la matriz son:");
        for (int i = 0; i < twoDimensionalArray.GetLength(0); i++)
        {
            for (int j = 0; j < twoDimensionalArray.GetLength(1); j++)
            {
                Console.Write(twoDimensionalArray[i, j] + " "); // mostrando los elementos de la fila separados por espacios especiales
                Console.WriteLine(); // salto de linea para mostrar los elementos de la siguiente fila
            }
        }

        Console.WriteLine("Los elementos de la matriz en forma de lista son:");
        for (int i = 0; i < twoDimensionalArray.GetLength(0); i++)
        {
            Console.Write("[");
            for (int j = 0; j < twoDimensionalArray.GetLength(1); j++)
            {
                Console.Write(twoDimensionalArray[i, j]);
                if (j != twoDimensionalArray.GetLength(1) - 1) Console.Write(", ");
            }
            Console.WriteLine("]"); // mostrando los elementos de la fila como una lista
        }
    }
}
